mbw.c

```diff
@@ -40,7 +44,7 @@
  * http://github.com/raas/mbw
  *
  * compile with:
- *			gcc -O -o mbw mbw.c
+ *			gcc -O -o mbw mbw.c -lpthread
```

Comment:
question: does the Makefile need updating (especially that the README.md says to use `make` to compile?)